frontend/src/components/HeroBanner.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const HeroBanner = () => {
+    return (
+        <BannerContainer>
+            <h1>Hero Banner</h1>
+            
+        </BannerContainer>
+    )
+}
+
+const BannerContainer = styled.div `
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 50vh;
+    background: url(https://www.enjoyintercambio.com/wp-content/uploads/2016/10/festival_cocktail_irlanda_enjoy_intercambio.jpg);
```

Comment:
Essa imagem está com 3.9MB, está muito pesada para ser renderizada em sites. Isso da muito diferença para quem tem uma internet um pouco ruim, e pior ainda para quem usa 4G. Até aqui para mim deu para perceber uma pequena demora no carregamento da imagem.

Geralmente da para usar imagens com tamanho até uns 300KB (talvez 200KB). Eu curto usar o site https://tinypng.com/ para diminuir o tamanho das imagens. Na parte free, ele não comprime tanto quanto poderia, mas já da uma grande melhorada nessa parte.

Aqui um exemplo dessa imagem comprimida com 295.6 KB:
![festival_cocktail_irlanda_enjoy_intercambio (1)](https://user-images.githubusercontent.com/54550926/140536342-6fa92289-beb9-4f10-8606-82fc0698fcf9.jpg)

Fiz o upload da imagem para o https://imgur.com/, e da para usar essa URL já com a imagem comprimida:
```suggestion
    background: url(https://i.imgur.com/tFklJI9.jpg);
```

Reply:
Da até para colocar essa imagem dentro da pasta `public` e chamar ela direto no código.

Exemplo, se o nome do arquivo for "banner.png", na pasta public (`/public/banner.png`), você consegue chamar no código assim: `/banner.png`.

Da para criar uma pasta "imagens" dentro da pasta `public`, para separar melhor as suas imagens, aí você usaria no código assim: `/imagens/banner.png`.